sansio_lsp_client/client.py

```diff
@@ -274,14 +431,163 @@ def did_close(self, text_document: TextDocumentIdentifier) -> None:
             params={"textDocument": text_document.dict()},
         )
 
-    def completions(
-        self,
-        text_document_position: TextDocumentPosition,
-        context: t.Optional[CompletionContext] = None,
+    def completion(
+            self,
+            text_document_position: TextDocumentPosition,
+            context: t.Optional[CompletionContext] = None,
     ) -> int:
         assert self._state == ClientState.NORMAL
         params = {}
         params.update(text_document_position.dict())
         if context is not None:
             params.update(context.dict())
         return self._send_request(method="textDocument/completion", params=params)
+
+# NEW #####################
+    def hover(
+            self,
+            text_document_position: TextDocumentPosition,
+            # WorkDoneProgressParams
```

Comment:
What does this comment mean?

Reply:
A reminder to add progress support, will remove those.

I don't plan to add that anymore